Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -398,9 +398,7 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
       .skipNil()
 
     self.dismissPersonalizationCell = self.personalizationCellDismissTappedProperty.signal
-      .on { _ in
-        AppEnvironment.current.userDefaults.hasDismissedPersonalizationCard = true
-      }
+      .on(value: { _ in AppEnvironment.current.userDefaults.hasDismissedPersonalizationCard = true })
```

Comment:
Interesting. The update to 6.5 wants the value param explicitly written now?

Reply:
It works without it but complains about ambiguity.